README.markdown

```diff
@@ -67,7 +66,7 @@ ini_subsetting {'sample subsetting':
 
 ###Use a non-standard section header
 
-~~~
+~~~puppet
 default:
    minage = 1
    maxage = 13
```

Comment:
This section is unclear; the first three lines are the resulting output of the following resource and should be broken into two code blocks, I think... maxage is no where to be seen, so should be removed from the output.